### PR TITLE
Implement UI design system

### PR DESCRIPTION
**Pull request by Mati0x**
#245

### CODE REVIEW

File: apps/web/styles/globals.css [hunk 1]
```diff
@@ -43,78 +60,119 @@ body {
   line-height: 1.15;
 }
 
-/* TODO: add Inter Font */
-
 h1 {
```
Comment (Corantin) [inline apps/web/styles/globals.css:63]: you can also customize components like this:
image
https://daisyui.com/docs/customize/

**Review by Corantin — CHANGES_REQUESTED** [review]

File: apps/web/tailwind.config.js [hunk 2]
```diff
@@ -24,13 +24,36 @@ module.exports = {
         chakra: ["var(--font-chakra)"],
       },
       colors: {
-        primary: "var(--color-primary)",
-        secondary: "var(--color-secondary)",
+        primary: "var(--color-green-500)",
```
[the remark this answers is not in the document]
Comment (Mati0x) [reply apps/web/tailwind.config.js:27]: We have only one theme momentary. Do you think it is mandatory for now goss ?

File: apps/web/styles/globals.css (hunk 1 above)
Comment (Mati0x) [reply apps/web/styles/globals.css:63]: Yes that an option we can adopt maybe for component level classNames ?
In this case we just customize a bit some basic html tags

File: apps/web/styles/globals.css [hunk 3]
```diff
@@ -43,78 +60,119 @@ body {
...
+  text-align: left;
+
+  /* @media only screen and (max-width: 680px) {
     font-size: 2rem;
-  }
+  } */
 }
 
 h3 {
   color: var(--color-black);
```
[the remark this answers is not in the document]
Comment (Mati0x) [reply apps/web/styles/globals.css:92]: We have to change h1 to h5 variable color yes, thx!

**Comment by Corantin** [comment]
I like the changes in the code 🔥
However now there is some problems in the app :S
image

**Comment by Mati0x** [comment]
> I like the changes in the code 🔥 However now there is some problems in the app :S image

Goss momentary yes, the app will change but I dont consider the app in problems. Along we start refactoring and building component and pages in the the app everything will look as our design system

**Review by Corantin — APPROVED** [review]

**Pull request merged**